### PR TITLE
update/fix deps

### PR DESCRIPTION
**Pull request by kevin-sakemaer**
- re-generate android and ios folder

this pack this open PR into one
- #136 
- #130 
- #129 

it also improve the ci

### CODE REVIEW

File: .fvm/flutter_sdk [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/kleak/fvm/versions/stable
```
Comment (jaumard) [inline .fvm/flutter_sdk]: not sure those should be in version source ^^

File: lib/flutter_camera_ml_vision.dart [hunk 2]
```diff
@@ -335,11 +329,8 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
   }
 
   Widget _getPicture() {
-    if (_lastImage != null) {
-      final file = File(_lastImage);
-      if (file.existsSync()) {
-        return Image.file(file);
-      }
+    if (_lastFrameFile != null && _lastFrameBytes != null) {
```
Comment (jaumard) [inline lib/flutter_camera_ml_vision.dart]: so you only need the bytes here, so `_lastFrameFile ` is not needed at all I would not keep a reference to it in that case

**Comment by kw2019ltd** [comment]
@jaumard hi is your fork working fine after updating camera plugin ?

**Comment by kw2019ltd** [comment]
dose it support null safety?

**Comment by kevin-sakemaer** [comment]
> dose it support null safety?
no we will probably do this upgrade in another PR.

keep in mind that with these version iOS is broken (at least when i did this PR) due to a problem in ml_vision package.

**Comment by kw2019ltd** [comment]
Sure iam using android only thanks will check it.

**Comment by jaumard** [comment]
Closing as 3.0.1 will contain all this :)

**Pull request closed without merging**